### ISSUE TITLE
IDISP025 doesn't emit with IAsyncDisposable

### ISSUE THREAD

**Issue by Kuinox** [DotNetAnalyzers/IDisposableAnalyzers#374]
> IDISP025: Class with no virtual dispose method should be sealed
```csharp
public class Diposable : IDisposable // IDISP025: Class with no virtual dispose method should be sealed
{
    public void Dispose()
    {
    }
}

public class AsyncDiposable : IAsyncDisposable // Nothing.
{
    public ValueTask DisposeAsync()
    {
        return new ValueTask();
    }
}
```

AFAIK there is no reason for not emitting this warning on AsyncDisposable.

**Comment by JohanLarsson** [comment]
Agreed analyzer should nag for consistency.
This may be an easy fix, could be analyzer code that is somewhat sane but could also be normal code.

**Comment by sorensenmatias** [comment]
I created a PR for this: https://github.com/DotNetAnalyzers/IDisposableAnalyzers/pull/416
